pre-commit/package_app_dependencies

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
https://askubuntu.com/a/503129

Reply:
this case can probably go either way (no extension or .sh), but the reason the other script also didn't have an extension is because the place I grabbed it from also had no extension :)